src/core/BasicCell/index.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-use-before-define */
+import { Args, Story } from "@storybook/react";
+import * as React from "react";
+import InputText from "../InputText";
+import BasicCellRaw from "./index";
+
+const BasicCell = (props: Args): JSX.Element => {
+  const [maxWidth, setMaxWidth] = React.useState(100);
+
+  const changeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setMaxWidth(+event.target.value);
```

Comment:
I haven't seen the notation with `+` before. what does that indicate?

Reply:
It is the same as `parseInt()`. The `event.target.value` is a type of string and `+` converts it to a number. 🥸